src/cafe/farm.py

```diff
@@ -5,12 +5,14 @@ class Farm:
     
     def __init__(self, farmerName:str='Farmer',
                  cuerdas:float=1,
-                 treeType:str='Borbon',
+                 treeType:str='Borbón',
```

Comment:
make sure 'borbon' as an argument behaves the same for those of us without convenient access to accents on keyboards ;D